sympy/series/tests/test_limits.py

```diff
@@ -1414,6 +1414,11 @@ def test_issue_26250():
     assert limit(e1/e2, x, 0) == -S(1)/8
 
 
+def test_issue_26513():
+    assert limit(abs((-n/(n+1))**n), n ,oo) == exp(-1)
+    raises (NotImplementedError, lambda: limit((-n/(n+1))**n, n, oo))
+
```

Comment:
I guess we should also add the tests which already work on master as control .  Can add all of these ig
```
    assert limit((x/(x + 1))**x, x, oo) == exp(-1)
    assert limit((-x/(x + 1))**x, x, oo) == Limit((-x/(x + 1))**x, x, oo)
    assert limit(abs((-x/(x + 1))**x), x, oo) == exp(-1)
    
```

Reply:
we cannot return an unevaluated limit for the second example u have stated as this roots from here
https://github.com/sympy/sympy/blob/95a8175eb35ca7c8d84afadafb8df4bfb93fd1d2/sympy/series/gruntz.py#L650-L653

Reply:
that was the whole point of the past pr not getting merged as we would then have to catch it here,
https://github.com/sympy/sympy/blob/95a8175eb35ca7c8d84afadafb8df4bfb93fd1d2/sympy/series/limits.py#L376-L381

Reply:
Yeah yeah was just pasting the examples here. Results might vary